### ISSUE TITLE
[BUG] - Change Data Folder Should Not Throw Errors When Selecting an Existing Folder 

### ISSUE THREAD

**Issue by mikeloeven** [ebkr/r2modmanPlus#823]
**Describe the bug**
Client throws non empty directory error when moving profile folder to another drive

**To Reproduce**
Copy Data Folder to New Location
Change Data Folder Location under settings to the new location

**Expected behavior**
Client will detect that the existing folder is a valid data folder and update the location without trying to recreate the folder structure

**Screenshots**
![image](https://user-images.githubusercontent.com/2266606/182970540-5d0bdaa8-d12a-4def-b881-6d9720a6278b.png)

**Comment by mikeloeven** [comment]
Did find a temp workaround by using Hard Link Shell Extension to create a symlink in the original location of the data folder pointing to the new location

**Comment by ValkierCS** [comment]
> **Describe the bug** Client throws non empty directory error when moving profile folder to another drive
> 
> **To Reproduce** Copy Existing Data Folder to New Location Change Data Folder Location under settings to the new location
> 
> **Expected behavior** Client will detect that the existing folder is a valid data folder and update the location without trying to recreate the folder structure
> 
> **Screenshots** ![image](https://user-images.githubusercontent.com/2266606/182970540-5d0bdaa8-d12a-4def-b881-6d9720a6278b.png)

Yup having the same issue I Had it twice and it's getting frustrating, I would hope they fix that in the near future or I will have to search for a New Mod manager that doesn't have that issue. It should not reset itself to change the location in the first place.

**Comment by ebkr** [comment]
Very delayed response but this should be fixed as of the latest manager version.
It does require having to set the directory one more time though, alternatively you can add a `.ddir.mm` file to the directory.
